src/components/ProgressBar/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react'
+
+import { Typography } from '../Typography'
+import s from './core/progressbar.module.scss'
+import { IProps } from './core/progressbar.typings'
+
+export const ProgressBar: FC<IProps> = ({ percentage }) => {
+  return (
+    <div className={s['container']}>
+      <Typography>{percentage}%</Typography>
+      <div className={s['progress-bar']}>
+        <div
+          className={s['progress-bar-active']}
+          style={{ width: `${percentage}%` }}
+        />
```

Comment:
Suggestion:
Use native and semantic [`<progress>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress) element and only tweak its style.

```suggestion
        <progress
          className={s['progress-bar-custom-styles-here']}
          max={props.max}
          value={props.value}
        />
```

Reply:
For custom styles of progress bar with semantic element `progress`, Firefox doesn't support this but we can try like this https://caniuse.com/?search=progress%20webkit :)